geometry_msgs/Point[] pixels
---
std_msgs/Header header
geometry_msgs/Point[] positions
